### PR TITLE
[SYCLomatic][DOC] Update DPCT1051 message in doc

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

File: docs/dev_guide/diagnostic_ref/dpct1051.rst [hunk 1]
```diff
@@ -8,9 +8,9 @@ Message
 
 .. _msg-1051-start:
 
-SYCL does not support the device property that would be functionally compatible
-with *<property name>*. It was migrated to *<migrated token>*. You may need
-to rewrite the code.
+SYCL does not support the device property functionally compatible with *<property name>*.
```
Comment (krisdale) [inline docs/dev_guide/diagnostic_ref/dpct1051.rst]: Minor edit (reads a little better): "SYCL does not support **a** device property functionally compatible ..."

File: docs/dev_guide/diagnostic_ref/dpct1051.rst [hunk 2]
```diff
@@ -8,9 +8,9 @@ Message
 
 .. _msg-1051-start:
 
-SYCL does not support the device property that would be functionally compatible
-with *<property name>*. It was migrated to *<migrated token>*. You may need
-to rewrite the code.
+SYCL does not support the device property functionally compatible with *<property name>*.
+It was migrated to *<migrated token>*, which value may need to be adjusted for
```
Comment (krisdale) [inline docs/dev_guide/diagnostic_ref/dpct1051.rst]: Update to: "It was migrated to \*<migrated token\>\*. You may need to adjust the value of \*<migrated token\>\* for the specific device."

**Review by krisdale — CHANGES_REQUESTED** [review]
Small edits for readability - approved with the edits applied.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**